homework.md

```diff
@@ -4,38 +4,38 @@
 
 What is 2 + 2?
 
-5
+The result is 4.
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+The opposite of black is white.
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+The result is 12240.
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+HTML stand for Hypertext Markup Language.
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+CSS helps us to change the style of web pages.
 
```

Comment:
You could expand this answer to include layout, organisation and colour of a website.